# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../edelta/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- edelta-0.9e/Makefile.vanilla	2007-03-09 09:17:09.000000000 +0000
+++ edelta-0.9e/Makefile	2007-03-09 09:17:55.000000000 +0000
@@ -1,3 +1,3 @@
-LDFLAGS = -lz
-CFLAGS = -Wall -ansi -finline-functions 
+LDLIBS = -lz
+CFLAGS = -Wall -finline-functions 
 edelta: edelta.c sha1.c
